app/services/deployment/server_actions/create.rb

```diff
@@ -3,21 +3,35 @@
 module Deployment
   module ServerActions
     class Create
-      def initialize(configurations)
+      def initialize(configurations, build_action)
         @configurations = configurations
+        @build_action = build_action
       end
 
       def call
-        @configurations.map do |configuration|
-          create_server(configuration)
-          push_code_to_server(configuration)
+        @configurations.each do |configuration|
+          deploy_configuration(configuration)
         rescue Excon::Error::UnprocessableEntity => error
+          logger.error(error.response.data[:body], context: configuration.application_name)
           puts JSON.parse(error.response.data[:body])
         end
       end
 
       private
 
+      attr_reader :build_action
+
+      def deploy_configuration(configuration)
+        app_name = configuration.application_name
+
+        create_server(configuration) && logger.info("Create server", context: app_name)
```

Comment:
I think it's better to log start of action:
`Creating server`
and then start this action, so during debugging we'll know which step goes wrong